### PR TITLE
formatting multilines

### PR DESCRIPTION
**Pull request by ninggao**

### CODE REVIEW

File: packages/dialog/templates/en-us/enumEntity.lg.lg [hunk 1]
```diff
@@ -3,24 +3,40 @@
 
 >>> This is because of issues around ' and } in strings--see enumEntityDesired.lg for a test case
 # template
-- @{propertyEntity()}\n@{chooseEntity()}\n\[library.lg]\n\[library-Choose.lg]
+-```
+ @{propertyEntity()}
+ @{chooseEntity()}
+ \[library.lg]\n\[library-Choose.lg]
```
Comment (chrimc62) [inline packages/dialog/templates/en-us/enumEntity.lg.lg]: >\[library.lg]\n\[library-Choose.lg] [](start = 1, length = 35)

Remove \n and I think you can get rid of the escaping \[

File: packages/dialog/templates/en-us/enumEntity.lg.lg [hunk 2]
```diff
@@ -3,24 +3,40 @@
...
 
 # chooseEntity
-- # choose@{property}Entity\n- @\{chooseEnumEntity(@{concat("'", property, "'")})\}\n
+-```
+ # choose@{property}Entity
+ - @\{chooseEnumEntity(@{concat("'", property, "'")})\}
+```
 
 # propertyEntity
-- @{join(createArray(namePlusSwitch(), cases(), default()), '\n')}\n
```
Comment (chrimc62) [inline packages/dialog/templates/en-us/enumEntity.lg.lg]: >\n [](start = 66, length = 2)

Did you look at the output?  Just wondering if you need  blank line in the ``` part.

File: packages/dialog/templates/en-us/library.lg [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 # requiredProperties
 - IF: @{count(dialogClass.schema.required) > 0}
-- Required properties include: @{join(foreach(dialogClass.schema.required, property, name(property)), ',')}\n
```
Comment (chrimc62) [inline packages/dialog/templates/en-us/library.lg]: >\n [](start = 107, length = 2)

Not sure about removing all of these.  Does it break tests?

File: packages/dialog/templates/en-us/library.lg [hunk 4]
```diff
@@ -45,7 +45,7 @@
 - Which property do you want to change?
 
 # notUnderstood
-- Sorry, I do not understand @{join(foreach(turn.unrecognizedtext, chunk, concat("'", chunk, "'")), ' or ')}\n
+- Sorry, I do not understand @{join(foreach(turn.unrecognizedtext, chunk, concat("'", chunk, "'")), ' or ')}
 
 > TODO: Would be good to split out all the component pieces, i.e. help, cancel, confirmation into their own files
 [library-PROPERTYName.lg]
```
Comment (chrimc62) [inline packages/dialog/templates/en-us/library.lg]: Remove this.

File: packages/dialog/templates/en-us/enumEntity.lg.lg (hunk 1 above)
Comment (ninggao) [reply packages/dialog/templates/en-us/enumEntity.lg.lg]: Done.

File: packages/dialog/templates/en-us/enumEntity.lg.lg (hunk 2 above)
Comment (ninggao) [reply packages/dialog/templates/en-us/enumEntity.lg.lg]: No, to make the output more compact, we don't need all those "\n".

File: packages/dialog/templates/en-us/library.lg (hunk 3 above)
Comment (ninggao) [reply packages/dialog/templates/en-us/library.lg]: The tests are fine.

File: packages/dialog/templates/en-us/library.lg (hunk 4 above)
Comment (ninggao) [reply packages/dialog/templates/en-us/library.lg]: I removed TODO.

**Pull request merged**